server/src/metadata/migration-runner/migration-runner.service.ts

```diff
@@ -113,13 +113,23 @@ export class MigrationGeneratorService {
             name: 'id',
             type: 'uuid',
             isPrimary: true,
-            default: 'uuid_generate_v4()',
+            default: 'public.uuid_generate_v4()',
           },
           {
             name: 'created_at',
             type: 'timestamp',
             default: 'now()',
           },
+          {
+            name: 'updated_at',
```

Comment:
Adding a few other default fields for the workspace tables, also we might need deleted_at later for soft deletion as I guess it will be the preferred approach